packages/react-composites/src/composites/CallComposite/utils/MediaGalleryUtils.test.tsx

```diff
@@ -0,0 +1,325 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { toFlatCommunicationIdentifier, _formatString } from '@internal/acs-ui-common';
+import { CallState, RemoteParticipantState } from '@internal/calling-stateful-client';
+import React, { useState } from 'react';
+import { COMPOSITE_LOCALE_EN_US } from '../../localization/locales';
+import { LocalizationProvider } from '../../localization/LocalizationProvider';
+import { CallAdapterProvider } from '../adapter/CallAdapterProvider';
+import { MockCallAdapter } from '../MockCallAdapter';
+import { useParticipantChangedAnnouncement } from './MediaGalleryUtils';
+import Enzyme, { ReactWrapper, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { act } from 'react-dom/test-utils';
+import { initializeIcons } from '@fluentui/react';
+import { CommunicationUserKind } from '@azure/communication-common';
+
+const strings = {
```

Comment:
Are these strings needed? Aren't these the same as the ones in the en-us string.json?